cryptofeed/backends/quasardb.py

```diff
@@ -0,0 +1,170 @@
+
+
+class BookQuasar(QuasarCallback, BackendCallback):
+    table_prefix = "book"
+
+    def format(self, data: dict):
+        data = super().format(data)
+        # store only best bid and best ask
+        if not data['book']:
+            best_bid = max(data["delta"]["bid"], key=lambda x: x[0])
+            best_ask = min(data["delta"]["ask"], key=lambda x: x[0])
+
+            data['best_bid_price'] = best_bid[0]
+            data['best_bid_amount'] = best_bid[1]
+            data['best_ask_price'] = best_ask[0]
+            data['best_ask_amount'] = best_ask[1]
+        else:
+            best_bid = max(data["book"]["bid"].keys())
+            best_ask = min(data["book"]["ask"].keys())
+
+            data['best_bid_price'] = best_bid
+            data['best_bid_amount'] = data["book"]["bid"][best_bid]
+            data['best_ask_price'] = best_ask
+            data['best_ask_amount'] = data["book"]["ask"][best_ask]
+        self.columns = list(data.keys())
+        self.columns.remove('book')
+        self.columns.remove('delta')
+        return data
+
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange SYMBOL(exchange), symbol SYMBOL(symbol), best_bid_price DOUBLE, best_bid_amount DOUBLE, best_ask_price DOUBLE, best_ask_amount DOUBLE, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
+
+
+class LiquidationsQuasar(QuasarCallback, BackendCallback):
+    table_prefix = "liquidations"
+
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange SYMBOL(exchange), symbol SYMBOL(symbol), side SYMBOL(side), quantity DOUBLE, price DOUBLE, id STRING, status STRING, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
+
+
+class OpenInterestQuasar(QuasarCallback, BackendCallback):
+    table_prefix = "open_intrerest"
+
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange SYMBOL(exchange), symbol SYMBOL(symbol), open_interest INT64, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
+
+
+class OrderInfoQuasar(QuasarCallback, BackendCallback):
+    table_prefix = "order_info"
+
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange SYMBOL(exchange), symbol SYMBOL(symbol), id STRING, client_order_id STRING, side SYMBOL(side), status STRING, type STRING, price DOUBLE, amount DOUBLE, remaining DOUBLE, account STRING, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
+
+
+class TransactionsQuasar(QuasarCallback, BackendCallback):
+    table_prefix = "transactions"
+
+    def _create_query(self):
+        self.query = f'CREATE TABLE "{self.table}" (exchange SYMBOL(exchange), currency SYMBOL(currency), type STRING, status STRING, amount DOUBLE, receipt_timestamp TIMESTAMP) SHARD_SIZE = {self.shard_size}'
```

Comment:
Couldn't / shouldn't `type` and `status` also be SYMBOL columns?

Same for teh other tables